src/components/i18nTextarea.vue

```diff
@@ -163,33 +163,21 @@ export default {
 				val: this.state,
 			})
 		},
-		focusOnLastTab: function() {
+		async focusOnLastTab() {
 			const last = Object.keys(this.value || {}).length - 1
-
-			// wait for tab to appear in DOM before switching to it;
-			// works for Chrome, but not Firefox
-			this.$nextTick(function() {
-				this.tabIndex = last
-				this.focusOnTextarea()
-			})
-
-			// In Firefox, for some reason, $nextTick only ever fires before the tab gets added,
-			// no matter how many times it gets called. So check again after a short delay.
-			setTimeout(() => {
-				if (this.tabIndex !== last) {
-					console.warn("setTimeout(): retry tab switch from", this.tabIndex, "to", last)
-					this.tabIndex = last
-					this.focusOnTextarea()
-				}
-			}, 300)
+			// bootstrap-vue takes a couple ticks to add a new tab to the DOM
+			await this.$nextTick()
+			await this.$nextTick()
```

Comment:
what if it takes three, four, five.. ticks?

Reply:
I am not pretty sure about this code. As Jori helped me here to fix browser related issue. so may be @tahme can explain this code.

Reply:
if you need to wait until something is in DOM, then why not check for that DOM element?

Reply:
The issue here is that typically when you change stuff, you have to wait one $nextTick until it appears in the DOM. The bootstrap-vue element we are using for the tabs also calls $nextTick before updating stuff, so we actually have to wait two ticks until the changes are reflected in the DOM. Two ticks should be enough but it's annoying that this depends on bootstrap-vue internals.

Reply:
ok, lets handle this in different issue in the future.